adaptor-rm-webapp/src/main/java/com/sample/rm/services/ServiceProviderService1.java

```diff
@@ -97,48 +101,12 @@
     @Context private HttpServletResponse httpServletResponse;
     @Context private UriInfo uriInfo;
 
+    private static final Logger log = LoggerFactory.getLogger(ServiceProviderService1.class);
```

Comment:
We need to show a big red error in LyoD instead of producing service1 service2 classes etc.